src/exception_utils.rs

```diff
@@ -171,6 +171,27 @@ pub fn exception_iss() -> usize {
     ESR_EL2.read(ESR_EL2::ISS) as usize
 }
 
+#[inline(always)]
+pub fn exception_sysreg_direction_write(iss: u64) -> bool {
+    const ESR_ISS_SYSREG_DIRECTION: u64 = 0b1;
+    (iss & ESR_ISS_SYSREG_DIRECTION) == 0
+}
+
+#[inline(always)]
+pub fn exception_sysreg_gpr(iss: u64) -> u64 {
+    const ESR_ISS_SYSREG_REG_OFF: u64 = 5;
+    const ESR_ISS_SYSREG_REG_LEN: u64 = 5;
+    const ESR_ISS_SYSREG_REG_MASK: u64 = (1 << ESR_ISS_SYSREG_REG_LEN) - 1;
+    (iss >> ESR_ISS_SYSREG_REG_OFF) & ESR_ISS_SYSREG_REG_MASK
+}
+
+#[inline(always)]
+pub const fn exception_sysreg_addr(iss: usize) -> usize {
+    // (Op0[21..20] + Op2[19..17] + Op1[16..14] + CRn[13..10]) + CRm[4..1]
```

Comment:
Please update the comment here.

Reply:
As in [here](https://github.com/arceos-hypervisor/aarch64_sysreg/blob/main/src/system_reg_type.rs).